cmd/start.go

```diff
@@ -34,30 +35,39 @@ func startCmd() *cobra.Command {
 			}
 			defer m.Close()
 
-			return runMigrationFromFile(cmd.Context(), m, fileName, complete)
+			c := backfill.NewConfig(
+				backfill.WithBatchSize(flags.BackfillBatchSize()),
+				backfill.WithBatchDelay(flags.BackfillBatchDelay()),
+			)
+
+			return runMigrationFromFile(cmd.Context(), m, fileName, complete, c)
 		},
 	}
 
+	startCmd.PersistentFlags().Int("backfill-batch-size", backfill.DefaultBatchSize, "Number of rows backfilled in each batch")
+	startCmd.PersistentFlags().Duration("backfill-batch-delay", backfill.DefaultDelay, "Duration of delay between batch backfills (eg. 1s, 1000ms)")
```

Comment:
Same comment about not needing to use `PersistentFlags` here as for the `migrate` command.